src/ui/buttons/buttonDesigns/divideLogoAnimation.jsx

```diff
@@ -141,6 +141,12 @@ export function getDivideLogoAnimation(designProps : DivideLogoAnimationProps, c
                 && paypalLabelContainerElement.contains(style)
             ) {
                 paypalLabelContainerElement.removeChild(style);
+            } else {
+                // enable animation again if size is between the expected range
+                if ((designContainer.offsetWidth < max && designContainer.offsetWidth > min)
```

Comment:
```
if (
    (designContainer.offsetWidth < max && designContainer.offsetWidth > min)
    && !paypalLabelContainerElement.contains(style)
) {
    paypalLabelContainerElement.appendChild(style);
}  
```
Could you do this instead. A bit more readable

Reply:
np